src/components/Feed/Feed.tsx

```diff
@@ -7,52 +7,63 @@ import './Feed.css';
 interface FeedProps {
   userSentiment: number | null;
   articles: CleanedArticle[];
+  updateHistory: (localHistory: string[]) => void;
   updateUserSentiment: (userSentiment: number) => void;
+  history: string[]
 }
 
-const Feed = ({ userSentiment, articles, updateUserSentiment }: FeedProps): JSX.Element => {
+const Feed = ({ userSentiment, articles, updateHistory, history, updateUserSentiment }: FeedProps): JSX.Element => {
+
+  const unreadArticles = articles.filter(article => {
+    return !history.includes(article.id)
+  })
 
   let sortedArticles : CleanedArticle[];
 
   if (userSentiment && userSentiment >= -1 && userSentiment <= -0.3) {
-    sortedArticles = articles.sort((articleA, articleB) => {
+    sortedArticles = unreadArticles.sort((articleA, articleB) => {
       return articleB.sentiment - articleA.sentiment;
     })
   } else if (userSentiment && userSentiment <= 1 && userSentiment >= 0.3) {
-    sortedArticles = articles.sort((articleA, articleB) => {
+    sortedArticles = unreadArticles.sort((articleA, articleB) => {
       return articleA.sentiment - articleB.sentiment;
     })
   } else {
-    sortedArticles = articles.sort((articleA, articleB) => 0.5 - Math.random());
+    sortedArticles = unreadArticles.sort((articleA, articleB) => 0.5 - Math.random());
   }
 
   const articleCards = sortedArticles.map(article => {
-    return  <Card
+    return  (
+      <Card
         title={ article.title }
         image={ article.multimedia.url }
         id={ article.id }
+        updateHistory={ updateHistory }
         sentiment={ article.sentiment }
         updateUserSentiment={ updateUserSentiment }
         key={ article.title }
       />
-    })
+    )
+  })
 
-    return (
-      <>
-        <div className="articles-container">
-          <Link to='/'>
-            <button className='retake-btn'>Retake Questionnaire</button>
-          </Link>
-          <Link to='/search-topic'>
-            <button className='search-topics-btn'>Search Topics</button>
-          </Link>
-            <section className="articles-display">
-              { articleCards }
-            </section>
-        </div>
-      </>
-    );
+  return (
+    <>
+      <div className="articles-container">
+        <Link to='/history'>
+          <button className='history-btn'>History</button>
+        </Link>
+        <Link to='/'>
+          <button className='retake-btn'>Retake Questionnaire</button>
+        </Link>
+        <Link to='/search-topic'>
+          <button className='search-topics-btn'>Search Topics</button>
+        </Link>
+          <section className="articles-display">
+            { articleCards.length ? articleCards : <h3>No more articles</h3> }
```

Comment:
When there are no more articles, I like that there is a message to the user. An idea would also be to include a Link to either the SearchTopic form so they can choose from another topic or to the history or both.

Reply:
I like this ideal!  Let me revisit after getting the buttons on the nav bar situated!  :)

Reply:
Good thinking. Added.